.eleventy.js

```diff
@@ -51,6 +51,8 @@ const tagsDir = 'src/site/_includes/components/tags';
 const {Image, Figure} = require(`./${tagsDir}/Image`);
 
 const collectionsDir = 'src/site/_collections';
+const pagedBlog = require(`./${collectionsDir}/paged-blog`);
```

Comment:
Maybe we should be even more explicit and name this collection `paginatedBlogPosts`.

For the other collection... we could call it something like `postsPaginatedByTag` ?